techeria_controller/templates/productInfo.html

```diff
@@ -0,0 +1,325 @@
+                    <ul class="image_list" style="list-style-type: none">
+                        <li> <img width="80%" src="/media/images/{{ oneProduct.image}}" alt=""></li>
+                        <li> <img width="80%" src="/media/images/{{ oneProduct.image}}" alt=""></li>
+                        <li> <img width="80%" src="/media/images/{{ oneProduct.image}}" alt=""></li> 
+                    </ul>
+                </div>
+                <div class="col-lg-4 order-lg-2 order-1">
+                    <div class="image_selected"><img width="70%" src="/media/images/{{ oneProduct.image}}" alt=""></div>
+                </div>
+                <div class="col-lg-6 order-3">
+                    <div class="product_description">
+                        <div class="product_name">{{ oneProduct.name }}</div>
+                        <div class="product-rating"> <a class="rating-review" href="#" style="color:black; text-decoration:underline;">Reviews</a></div> <br>
+                        <div> <span class="product_price">${{ oneProduct.price }}</span> </div>
+                        <hr class="singleline">
+                        <div> <span class="product_info">{{ oneProduct.description }}</span></div>
+                        <hr class="singleline">
+                        <div class="row">
+                            <div class="col-xs-6" style="margin-left: 13px;">
+                                <div class="product_quantity"> <span>QTY: </span> <input id="quantity_input" type="text" pattern="[0-9]*" value="1">
+                                    <div class="quantity_buttons">
+                                        <div id="quantity_inc_button" class="quantity_inc quantity_control"><i class="fas fa-chevron-up"></i></div>
+                                        <div id="quantity_dec_button" class="quantity_dec quantity_control"><i class="fas fa-chevron-down"></i></div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="col-xs-6"> <button type="button" class="btn btn-primary shop-button">Add to Cart</button> <span style="display:inline-block; width: 10px;"></span> <button type="button" class="btn btn-success shop-button">Buy Now</button>
+                                <span style="display:inline-block; width: 10px;"></span>
+                                <div <button type="button"class="product_fav"><i class="fas fa-heart"></i></button></div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <br>
+            <br>
+            <br>
+    
+
+  <!-- footer section -->
+
+  <script type="text/javascript" src=" {% static 'js/jquery-3.4.1.min.js' %} "></script>
+  <script type="text/javascript" src=" {% static 'js/bootstrap.js' %} "></script>
+</body>
+
+  <section class="container-fluid footer_section">
+    <p>
```

Comment:
the footer looks different. the footer has to be changed. since we have decided on the product page footer to follow.